1-estrategia/README.md

```diff
@@ -2,22 +2,129 @@
 
 ## Objetivos 
 
+**Producto**
+
+Es una plataforma web informativa denominada *Aupa "La plataforma que te ayuda"*, consiste en ordenar la información relevante de  instituciones publicas y ONGs que trabajan para integrar a los migrantes forzados por cuestiones políticas, económicas, guerras (...). Eventualmente, crearemos contenido que complemente lo anterior para cumplir los objetivos del producto. 
+
+
 ### Visión
 
+Ser referentes número uno para todos los refugiados e inmigrantes forzados que llegan al País Vasco para que tengan una correcta inserción dentro de la comunidad.
+
+__*¿Cómo veo mi producto?*__
+
+Queremos que Aupa sea un sitio web donde los migrantes puedan encontrar información útil para la acogida en el País Vasco, Además será una plataforma que les permita interactuar y contar sus experiencias al mismo tiempo que se van adaptando al entorno. 
+
+__*¿Cómo quiero que sea el producto?*__
+
+El producto está pensado como una línea de tiempo.
+Nos referimos a una secuencia de pasos que tiene como punto de partida el momento en que los inmigrantes llegan al País Vasco, desde ahí, se da inicio a una serie de pasos que al ser completados, puedan lograr una plena inserción dentro de la comunidad.
+
+![Los pasos](https://github.com/DeustoPWEB2018/proyectoweb-migraciones/blob/85faae59c23aa3f2042f300a18d008d457ecc177/1-estrategia/pasos.png?raw=true)
+
+__*¿Cuál es el propósito de mi producto?*__
+
+Buscamos ser ese facilitador para los inmigrantes forzados que pueda brindarle un marco formal ó camino a seguir para resolver sus necesidades e inquietudes con el menor plazo de tiempo posible.
+De un modo simplificado, partiremos dando por sentado una situación común a todos los inmigrantes forzados la que denominaremos "punto cero" con la siguiente característica: inmigrante forzado que llega al Pais Vasco sin tener resuelto de antemano los papeles migratorios, alojamiento, alimentación, educación, trabajo, atención sanitaria, red de vínculos sociales, etc. Desde allí, existirán "x pasos", que prevén, facilitar el camino a seguir para la resolución ó superación de todas las necesidades que tienen hasta alcanzar el paso final que les permita la plena inserción en la comunidad vasca.
```

Comment:
```suggestion
De un modo simplificado, partiremos dando por sentado una situación común a todos los inmigrantes forzados la que denominaremos "punto cero" con la siguiente característica: inmigrante forzado que llega al País Vasco sin tener resueltos de antemano los papeles migratorios, alojamiento, alimentación, educación, trabajo, atención sanitaria, red de vínculos sociales, etc. Desde allí, existirán "x pasos", que prevén, facilitar el camino a seguir para la resolución ó superación de todas las necesidades que tienen hasta alcanzar el paso final que les permita la plena inserción en la comunidad vasca.
```

Sin saber mucho, anticipo que tenéis mucho trabajo definiendo qué es exactamente la _plena inserción_. Por plantear una dificultad muy simple, ¿os dirigís a todos los migrantes independientemente de si tienen o no intención de quedarse en el País Vasco? ¿Os parece que tienen las mismas necesidades de información los migrantes en tránsito de quienes "vienen para quedarse"?

Reply:
@angielozanogr he anadido "Esta permitiria a los inmigrantes que quieren quedarse en el Pais Vasco de se integrar. Por ejemplo, buscar un trabajo, obtener la tarjeta de salud, buscar un alojamiento. "